src/components/general/EntryVerify/index.tsx

```diff
@@ -117,16 +121,11 @@ function EntryVerify(props: Props) {
         setEntryVerificationRequest,
     } = requests;
 
-    // FIXME: we can remove verified and use value instead
-    const [verified, setVerificationStatus] = useState(value);
-
-    useEffect(() => {
-        setVerificationStatus(value);
-    }, [value]);
-
-    // FIXME: memoize this
-    const selectedOption = verificationStatusOptions.find(v => v.key === verified) ||
-                           verificationStatusOptions[1];
+    const selectedOption = useMemo(
+        () => verificationStatusOptions.find(v => v.key === value) ||
+            verificationStatusOptions[1],
```

Comment:
```suggestion
        () => verificationStatusOptions.find(v => v.key === value) ??
            verificationStatusOptions[1],
```

Also why not use `0`?

Reply:
0 is for verified

Reply:
I think we should not hard-code this in this way. 
Thoughts @frozenhelium @AdityaKhatri ?

Reply:
Not ideal, but works right now

Reply:
"Works right now" doesn't work for me.